scrapers/sd/bills.py

```diff
@@ -163,66 +159,97 @@ def scrape_action(self, bill, url, chamber):
                     self.error("Unknown vote code: {}".format(action["Result"]))
                 atypes.append("%s%s" % (first, second))
 
+            if (
+                action_text == "Do Pass"
+                or action_text == "Tabled"
+                or "ShowCommitteeName" in action
+            ):
+                full_action.insert(0, f'{action["ActionCommittee"]["Name"]}')
```

Comment:
Let's filter out "House of Representatives" and "Senate" from this, since the chamber is mentioned in the action anyway.

Reply:
Switching `or "ShowCommitteeName" in action` to `or action["ShowCommitteeName"]` does this, I just mis-interpreted converting the site logic. Should be fixed now.

Reply:
thanks for this fix, the rest looks good from here!